lib/modules/manager/fleet/extract.ts

```diff
@@ -50,6 +52,22 @@ function extractFleetHelmBlock(doc: FleetHelmBlock): PackageDependency {
       skipReason: 'missing-depname',
     };
   }
+
+  if (isOCIRegistry(doc.chart)) {
+    const ociImageRegistry = doc.chart.replace('oci://', '');
+    const dockerDep = getDep(`${ociImageRegistry}:${doc.version}`, false);
+
+    return {
+      ...dockerDep,
+      depType: 'fleet',
+      depName: ociImageRegistry.split('/').at(-1),
```

Comment:
I think we should use the full name here, otherwise it can be confusing

Reply:
Based on the test:

```yaml
defaultNamespace: external-dns
helm:
  chart: oci://registry-1.docker.io/bitnamicharts/external-dns
  version: 7.1.2
```

The current iteration would set depName to "external-dns". Which would be the full name?

- `oci://registry-1.docker.io/bitnamicharts/external-dns`?
- `registry-1.docker.io/bitnamicharts/external-dns`?
- `bitnamicharts/external-dns`?

Reply:
Use `registry-1.docker.io/bitnamicharts/external-dns`